### PR TITLE
Multiply return time by 100 in tests to detect off by one differences

### PR DESCRIPTION
**Pull request by serathius**
cc @ah8ad3 @ahrtr @MadhavJivrajani @henrybear327 @siyuanfoundation @jmhbnz

### CODE REVIEW

**Comment by serathius** [comment]
cc @fuweid

**Review by fuweid — APPROVED** [review]

File: tests/robustness/validate/patch_history_test.go [hunk 1]
```diff
@@ -40,263 +42,263 @@ func TestPatchHistory(t *testing.T) {
...
+				h.AppendPutWithLease("key2", "value", 234, 300, 400, &clientv3.PutResponse{}, nil)
 			},
 			persistedRequest: []model.EtcdRequest{
 				putRequestWithLease("key1", "value", 123),
 				putRequestWithLease("key2", "value", 234),
 			},
 			expectedRemainingOperations: []porcupine.Operation{
-				{Return: 3, Output: model.MaybeEtcdResponse{Persisted: true}},
-				{Return: 4, Output: putResponse(model.EtcdOperationResult{})},
+				{Return: 399, Output: model.MaybeEtcdResponse{Persisted: true}},
```
Comment (jmhbnz) [inline tests/robustness/validate/patch_history_test.go:152]: Just want to double check this is intentional for `399` rather than `300`?

**Comment by serathius** [comment]
ping @jmhbnz

File: tests/robustness/validate/patch_history_test.go (hunk 1 above)
Comment (jmhbnz) [reply tests/robustness/validate/patch_history_test.go:152]: Thanks for the explanation, makes sense to me now.

**Pull request merged**